src/index.ts

```diff
@@ -58,6 +58,7 @@ export class Probot {
     switch (err.message) {
       case 'X-Hub-Signature does not match blob signature':
       case 'No X-Hub-Signature found on request':
+      case: 'webhooks:receiver ignored: POST / due to missing headers: x-hub-signature':
```

Comment:
@andymckay's comment about the `:` beside `case` was spot on - that's why CI is failing.